app/src/main/java/org/gdg/frisbee/android/api/OkClientFactory.java

```diff
@@ -45,10 +45,13 @@ public void log(String message) {
         return okHttpClient.build();
     }
 
-    public static OkHttpClient okHttpClientWithIdlingResources(OkHttpClient client) {
-        return client.newBuilder()
-            .addInterceptor(OkClientFactory.provideIdlingResourcesInterceptor())
-            .build();
+    public static OkHttpClient okHttpClientWithIdlingResources(OkHttpClient client, Interceptor ... interceptors) {
```

Comment:
The name of this method is now weird. It adds both idlingResources and any Interceptor given. 
I think these should be done separately.